packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -89,13 +89,21 @@ Future<io.WebSocket> _defaultOpenChannel(String url, {
   io.WebSocket socket;
 
   Future<void> handleError(dynamic e) async {
-    globals.printTrace('Exception attempting to connect to Observatory: $e');
-    globals.printTrace('This was attempt #$attempts. Will retry in $delay.');
-
+    void Function(String) printVisibleTrace = globals.printTrace;
     if (attempts == 10) {
-      globals.printStatus('This is taking longer than expected...');
+      globals.printStatus('Connecting to the Observatory is taking longer than expected...');
```

Comment:
While the previous message did say observatory, let's update these new message to specify "VM Service", which is more correct

Reply:
Done.